plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/JobIsRunning.java

```diff
@@ -74,8 +96,13 @@ public boolean test() {
 				continue;
 			}
 
-			if (job.isSystem() || job.getState() == Job.SLEEPING) {
-				log.debug("  job '%s' is system job or not running, skipped", job.getName());
+			if (skipSystemJobs && job.isSystem()) { 
+				log.debug("  job '%s' is a system job, skipped", job.getName());
+				continue;
+			}
+			
+			if (job.getState() == Job.SLEEPING) {
```

Comment:
Shouldn't there be else ? For system job which is sleeping I will get two log messages which is not necessary